src/partition.rs

```diff
@@ -718,6 +718,7 @@ pub const GLOBALMV_CTX_MASK: usize =
   ((1 << (REFMV_OFFSET - GLOBALMV_OFFSET)) - 1);
 pub const REFMV_CTX_MASK: usize = ((1 << (8 - REFMV_OFFSET)) - 1);
 
+// Caution: PARTITION_SPLIT must be the last entry!
```

Comment:
A unit test would be good to assert this.

Reply:
Right, I will do that-

Reply:
This comment is probably no longer needed, since the unit test documents the need for `PARTITION_SPLIT` to be last.

Reply:
Would others agree on rzumer's comment? I'd prefer keep it there since many of us not checking what's in the test.